### DISCUSSION TITLE
프로필 불러오는 기능 및 테스트 월드 오류

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#85] [5. Q&A]
안녕하세요,
현재 테스트월드 오류가 있는 것처럼 접속이 안되고 있습니다. 그리고 제페토 프로필 사진을 불러오는 기능도 됐다 안됐다 합니다. 확인 부탁드리겠습니다.

**Comment by cheolhui-lee** [comment]
안녕하세요.
서버 이슈는 조금 전 수정이 되었는데 혹시 여전히 안되고 있으신지 확인 부탁드려도 될까요?

**Reply by diddntjd99** [reply]
정상 작동되는 것 같습니다
